src/test/java/org/folio/marccat/resources/AuthorityRecordTest.java

```diff
@@ -22,15 +25,18 @@
 @ActiveProfiles("test")
 public class AuthorityRecordTest extends TestBase {
 
+  private static final String CONTENT_TYPE = "Content-Type";
+  private static final String FILE_TYPE = "application/json";
+
   @Test
-  public void save_return201Status() throws IOException {
+  public void saveReturn201Status() throws IOException {
     String url = getURI("/marccat/authority-record");
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
     String templateJson = IOUtils.toString(this.getClass().getResourceAsStream("/authority/name.json"),
         String.valueOf(StandardCharsets.UTF_8));
 
-    given().headers("Content-Type", "application/json").headers(headers).queryParam("view", "-1")
-        .queryParam("lang", "eng").body(templateJson).when().post(url).then().statusCode(201);
+    given().headers(CONTENT_TYPE, FILE_TYPE).headers(headers).queryParam("view", "-1").queryParam("lang", "eng")
+        .body(templateJson).when().post(url).then().statusCode(201);
```

Comment:
This is really a style preference, but I see most teams splitting this up as it's easier to read... something like:
```suggestion
    given()
        .headers(CONTENT_TYPE, FILE_TYPE)
        .headers(headers)
        .queryParam("view", "-1")
        .queryParam("lang", "eng")
        .body(templateJson)
    .when().post(url)
        .then().statusCode(201);
```

Reply:
I agree too

Reply:
I have changed it